docs/input/docs/usage/obtain.md

```diff
@@ -0,0 +1,9 @@
+---
+Order: 10
+Title: Obtain
+Description: Obtain BBT.Maybe
+---
+
+## Nuget
```

Comment:
See https://github.com/bbtsoftware/BBT.StructureTools/pull/24 for a suggested improvement of this page

Reply:
Done.